INN/umbrella-workdayminn
# 9 fonts and pull quotes

## Changes

This pull request makes the following changes:

- adds fonts, for #7 
- updates colors in stylesheet to match the values given in the mockup linked in #7
- adds those colors as the pre-selected colors in the Gutenberg color chooser
- pull quote styles for #9
    - different styles for the "solid color background" and "standard" style variants of Gutenberg blocks
    - orange "" for the solid background, padding so that it can have colored backgrounds with the black borders inside the background
    - green "" for the plain background, no padding
    -  10% outwards margin offset for alignleft, alignright.

![Screen Shot 2019-07-31 at 20 20 58 ](https://user-images.githubusercontent.com/1754187/62256989-860c4a80-b3d1-11e9-9fac-ec1ba4a3c0b1.png)
![Screen Shot 2019-07-31 at 20 14 46 ](https://user-images.githubusercontent.com/1754187/62256997-8c9ac200-b3d1-11e9-8cfa-77939004a1c8.png)
![Screen Shot 2019-07-31 at 17 13 07 ](https://user-images.githubusercontent.com/1754187/62257004-93c1d000-b3d1-11e9-844c-3b78a7fc2d1d.png)

## Testing/Questions

Features that this PR affects:

- the pull quote block, in all varieties
- Gutenberg color picker
- fonts across the site


Questions that need to be answered before merging:

- [x] fonts look okay?
- [x] blocks look okay?
- [ ] mockup didn't have a style for quote citations; should those be styled differently from the paragraphs in the quote?

Steps to test this PR:

1. create left/right/none/center/wide blocks, in the "Default" and "Solid Color" style variants
2. view at different sizes
